docs/app/pages/components/components-sections/tables/filters/filters.component.ts

```diff
@@ -13,6 +13,9 @@ import { IPlaygroundProvider } from '../../../../../interfaces/IPlaygroundProvid
 @DocumentationSectionComponent('ComponentsFiltersComponent')
 export class ComponentsFiltersComponent extends BaseDocumentationSection implements IPlaygroundProvider {
 
+    colorTheme: any;
```

Comment:
We don't want to use `any`, however these changes can probably be removed after my comments above